src/app/App.tsx

```diff
@@ -1,13 +1,20 @@
 import React from "react";
 import styles from "./App.module.css";
 import { BrowserRouter, Switch, Route, RouteProps } from "react-router-dom";
-import Search from "./pages/Search/Search";
+import Search from "./pages/Pages/Search/Search";
+import Home from "./pages/Pages/Home/Home";
 
 type CustomRouteProps = RouteProps & {
   Component: () => JSX.Element;
   path: string;
 };
-const routes: CustomRouteProps[] = [{ path: "/search", Component: Search }];
+const routes: CustomRouteProps[] = [
```

Comment:
vielleicht kannst du für die Zukunft drüber nachdenke die routes in ne eigene file auszulagern